app/views/partials/backup_code/_entry_fields.html.slim

```diff
@@ -3,7 +3,7 @@
     label: t('simple_form.required.html') + t('forms.two_factor.backup_code'),
     input_html: { autocapitalize: 'none',
       autocomplete: 'off',
-      class: 'caps',
+      class: 'caps backup_code',
```

Comment:
for consistency with `personal-key` and others, WDYT of using dashes? `backup-code`?

Reply:
yes, thank you!